samples/ingestion/ingestion-client/FetchTranscription/Language/AnalyzeConversationsProvider.cs

```diff
@@ -0,0 +1,255 @@
+﻿// <copyright file="AnalyzeConversationsProvider.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace FetchTranscription.Language
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+    using Azure;
+    using Azure.AI.Language.Conversations;
+    using Azure.Core;
+    using Connector;
+    using Connector.Serializable.Language.Conversations;
+    using FetchTranscriptionFunction;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Analyze Conversations async client.
+    /// </summary>
+    public class AnalyzeConversationsProvider
+    {
+        private static readonly TimeSpan RequestTimeout = TimeSpan.FromMinutes(3);
+
+        private readonly ConversationAnalysisClient ConversationAnalysisClient;
+
+        private readonly string Locale;
+
+        private readonly ILogger Log;
+
+        public AnalyzeConversationsProvider(string locale, string subscriptionKey, string region, ILogger log)
+        {
+            ConversationAnalysisClient = new ConversationAnalysisClient(new Uri($"https://{region}.api.cognitive.microsoft.com"), new AzureKeyCredential(subscriptionKey));
+
+            Locale = locale;
+            Log = log;
+        }
+
+        public bool IsConversationalPiiEnabled()
+        {
+            return Locale.Contains("en", StringComparison.OrdinalIgnoreCase) && FetchTranscriptionEnvironmentVariables.ConversationPiiSetting != Connector.Enums.ConversationPiiSetting.None;
```

Comment:
Keep in mind that you can't update the ingestion client deployments, so customers would need to re-deploy their solution if you're planning to add new locales.
Maybe it'd be better to just let the TA request fail?

Reply:
thats fair.